draft-ietf-ppm-dap.md

```diff
@@ -1034,13 +1041,11 @@ DAP has three major interactions which need to be defined:
 * The Collector collects aggregated results from the Aggregators, specified in
   {{collect-flow}}
 
-Each of these interactions is defined in terms of "resources". In this section
-we define these resources and the messages used to act on them.
+Each of these interactions is defined in terms of HTTP resources. In this
+section we define these resources and the messages used to act on them.
```

Comment:
Avoid scare quotes and make explicit reference to HTTP, which has a strong concept of a [resource](https://datatracker.ietf.org/doc/html/rfc9110#section-3.1).